src/content/docs/en/reference/container-reference.mdx

```diff
@@ -376,3 +376,20 @@ const response = await container.renderToResponse(Endpoint, {
 });
 const json = await response.json();
 ```
+
+### `partial` option
+
+**Type**: `boolean`
```

Comment:
```suggestion
<p>

**Type:** `boolean`<br />
**Default:** `true`<br/>
<Since v="4.16.6" />
</p>
```

(I think that will be the next version!)